ambuda/templates/macros/library.html

```diff
@@ -2,9 +2,9 @@
 
 {% macro text_tabs(text, active) %}
 {% set routes = [
-  ("contents",  "Contents",  url_for("texts.text", slug=text.slug)),
-  ("resources", "Resources", url_for("texts.text_resources", slug=text.slug)),
-  ("about",     "About",     url_for("texts.text_about", slug=text.slug)),
+  ("contents",  _("Contents"),  url_for("texts.text", slug=text.slug)),
+  ("resources", _("Resources"), url_for("texts.text_resources", slug=text.slug)),
+  ("about",     _t("About"),     url_for("texts.text_about", slug=text.slug)),
```

Comment:
Why _t("foo") instead of _("foo")?  Adding some guidance in [docs](https://github.com/ambuda-org/ambuda/pull/332/files#diff-2e4cd15dd754e51fc2b7063651bc274e0e6aeae9133b592ce1774f933db703f2)  on _t usage might be useful